sample-app/src/main/java/rx/android/samples/MainActivity.java

```diff
@@ -19,7 +19,7 @@
 public class MainActivity extends Activity {
     private static final String TAG = "RxAndroidSamples";
 
-    private Handler backgroundHandler;
+    private Handler mBackgroundHandler;
```

Comment:
Maybe this should be added to a coding style guide? For RxAndroid (the *Android specific* bindings for RxJava) I assumed the standard Android naming conventions would apply.